agasc/agasc.py

```diff
@@ -353,4 +361,42 @@ def get_stars(ids, agasc_file=None, dates=None, fix_color1=True):
         update_color1_column(t)
     t['DATE'] = dates
 
+    t = _add_supplement(t, supplement_file)
+
     return t if ids.shape else t[0]
+
+
+def _add_supplement(table, supplement_file=None):
```

Comment:
For performance:
- The supplement values should be made as a module-level LazyDict. We really need to put this into some common utilities package since it will now be in three places, most recently https://github.com/sot/eng_archive/pull/206.  `ska_helpers`?
- Instead of astropy table join, given the sparse nature of coverage (typically zero to a handful out of hundreds of stars in a field will have data), best to create the columns in advance by copying `MAG_ACA` to `MAG_EST` (and same for the error). Then just loop over the AGASC_ID values and if the id is in the supplement dict then overwrite accordingly.